projects/storefrontlib/src/cms-components/organization/order-approval/order-approval-list/order-approval-list.component.html

```diff
@@ -0,0 +1,32 @@
+<cx-interactive-table
+  [data$]="data$"
+  [header]="'orderApprovalsList.header' | cxTranslate"
+  [sortLabels]="{
+    byOrderNumber: 'orderApprovalsList.sorting.byOrderNumber' | cxTranslate,
+    byDate: 'orderApprovalsList.sorting.byDate' | cxTranslate
+  }"
+  [columns]="[
+    {
+      key: 'code',
+      value: 'orderApprovalsList.code' | cxTranslate,
+      cxRoute: 'orderApprovalsDetails'
```

Comment:
The route for approval details is `orderApprovalDetails`.

The approval details needs the order approval code.  
```
/my-account/approval/{approvalCode}
```
Is the approval code passed to the details page?